### ISSUE TITLE
Assuming three years old version 999 to be most recent

### ISSUE THREAD

**Issue by jacobqvist** [coana-tech/coana-issue-tracker#30]
While updating my dependencies in a Nx Angular project I saw the following:

![image](https://github.com/coana-tech/coana-issue-tracker/assets/5192705/14232baa-62fa-4f9d-8bd0-daa3514072ac)

Don't know the reason why they would ever make a release with a 999 and a 9999 major version. Version 9999 was released three years ago. You might want to look into this?

**Comment by mtorp** [comment]
Hi @jacobqvist,

Thanks for reporting this!

Actually, the version does exist:

```
$ npm info nx versions | tail -n 2
  '16.4.0-beta.0',  '16.4.0-beta.1',  '999.9.9',        '9999.0.1'
]

```

However, it is marked as deprecated, so I agree that we shouldn't show it in the list. I'll look into this ASAP.

**Comment by mtorp** [comment]
I've just released version 0.18.1 which fixes this issue.

Versions now either shown as deprecated if they are explicitly deprecated or as beta if the are above the version that is tagged as latest.

Thanks again for reporting this issue @jacobqvist
